spec/ParseUser.spec.js

```diff
@@ -4447,55 +4447,41 @@ describe('custom log levels for username already exists error', () => {
     infoSpy.calls.reset();
   });
 
-  it('should respect warn log level for username already exists error', async () => {
-    await reconfigureServer({
-      logLevels: {
-        usernameAlreadyExists: 'warn',
-      },
-    });
+  const testCases = [
+    { logLevel: 'warn', expectedSpy: 'warnSpy' },
+    { logLevel: 'info', expectedSpy: 'infoSpy' },
+  ];
+
+  testCases.forEach(({ logLevel, expectedSpy }) => {
+    it(`should respect ${logLevel} log level for username already exists error`, async () => {
```

Comment:
This is a structure that we do not support, i.e. looping tests themselves. Could you please refactor this info 1 single test and run a loop inside the test?